traffic_ops/app/lib/API/Deliveryservice.pm

```diff
@@ -909,8 +945,13 @@ sub capacity {
 	my $id = $self->param('id');
 
 	if ( $self->is_valid_delivery_service($id) ) {
-		if ( $self->is_delivery_service_assigned($id) || &is_admin($self) || &is_oper($self) ) {
+		my $tenant_utils = Utils::Tenant->new($self);
+		my $tenants_data = $tenant_utils->create_tenants_data_from_db();
+		if ( $self->is_delivery_service_assigned($id) || $tenant_utils->ignore_ds_users_table() || &is_admin($self) || &is_oper($self) ) {
```

Comment:
i don't "think" this makes sense

if $tenant_utils->ignore_ds_users_table() is true then it doesn't matter if  is_delivery_service_assigned($id), does it? also, calling both &is_admin($self) || &is_oper($self) is redundant. &is_oper($self) covers both is ops and is admin